### PR TITLE
Fix robots.txt

### PR DESCRIPTION
**Pull request by remyleone**
Lighthouse in Google Chrome reported an issue

### CODE REVIEW

File: layouts/robots.txt [hunk 1]
```diff
@@ -4,6 +4,6 @@ Disallow: /legacy/
 Disallow: /v1.0/
 Disallow: /v1.1/
 Disallow: /404/
-Disallow: 404.html
+Disallow: /404.html
 
-SITEMAP: {{ "sitemap.xml" | absLangURL }}
\ No newline at end of file
+SITEMAP: {{ "sitemap.xml" | absLangURL }}
```
Comment (kbhawkey) [inline layouts/robots.txt]: Is there a difference between the SITEMAP entries?

**Comment by remyleone** [comment]
No just a new line at the end

Em sex, 7 de jun de 2019 19:19, Karen Bradshaw <notifications@github.com>
escreveu:

**Pull request merged**